src/components/video-player/video-player.jsx

```diff
@@ -0,0 +1,31 @@
+import React, {PureComponent, createRef} from "react";
+import PropTypes from "prop-types";
+
+class VideoPlayer extends PureComponent {
+  constructor(props) {
+    super(props);
+
+    this._videoRef = createRef();
+
+    this.state = {
+      isMuted: props.isMuted,
+      isPlaying: props.isPlaying
+    };
+  }
+  render() {
+    const isMuted = this.state.isMuted;
+    const isPlaying = this.state.isPlaying;
+    return (
+      <video autoPlay={isPlaying} muted={isMuted} ref={this._videoRef} src={this.props.video} className="player__video" poster={this.props.preview}/>
```

Comment:
длинные строки лучше разбивать
```suggestion
      <video 
        autoPlay={isPlaying} 
        muted={isMuted} 
        ref={this._videoRef} 
        src={this.props.video} 
        className="player__video" 
        poster={this.props.preview}
      />
```